### ISSUE TITLE
Locked out of CDD, HSTS preventing me from getting to web UI

### ISSUE THREAD

**Issue by joshstrange** [githubsaturn/captainduckduck#348]
I am locked out of the web admin for CDD. I noticed this when I visited one of the sites I host on CDD and got a certificate expired error. I would have thought that CDD would auto-renew these for me but it's not the end of the world so I went to the CDD web UI to refresh it or try to do something about it. That's when I got the same (or so I thought) error. No surprise, if CDD didn't renew my sites I wouldn't expect it to renew itself so I go to bypass the warning so I can get everything fixed... That's when I saw the dreaded HSTS error...

So I currently can't get to the web UI for CDD. Is there a CLI command I can run to fix this or something? I (obviously) have ssh access to this server still but I'm not sure what to do to fix this and I don't want to break it further.

Stretch Goal Question: Is there a way to setup auto-renewing of TLS certs?

**Comment by joshstrange** [comment]
Looks like #259 might be what I need to do, I'll attempt a restart

**Comment by joshstrange** [comment]
A [restart](https://captainduckduck.com/docs/troubleshooting.html#how-to-restart-captainduckduck) fixed this issue. Thank you. I would find #259 only after other fruitless searches and posting here.
